mypy/errors.py

```diff
@@ -808,20 +801,26 @@ def save_baseline(self, file: Path) -> None:
         if not file.parent.exists():
             file.parent.mkdir()
         json.dump(
-            filter_prefix(self.error_info_map),
+            {
+                file: [
+                    {
+                        "line": error.line,
+                        "code_description": error.code.description,
```

Comment:
why use the description instead of just the code?

Reply:
good point

Reply:
baseline still has descriptions in it, need to regenerate it?